moodle/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+# This Docker compose file is written to configure services required to deploy Moodle, e-learning platform,
+# on containers
+# Author: Zainab Alsaffar
+version: '3'
+volumes:
+  mariadb-data:
+    driver: local
+  moodle-data:
+    driver: local
+networks:
+    moodle-network:
+services:
+  mariadb:
+    image: 'mariadb:latest'
+    container_name: mariadb
+    environment:
+      MYSQL_ROOT_PASSWORD: YOUR_PASSWORD
```

Comment:
Could we follow the same pattern for env vars that I have in the elgg docker compose? (`env_file`)

Thanks!

Reply:
@thatnerdjosh , Sure! will try that out.